config/initializers/rollbar.rb

```diff
@@ -0,0 +1,10 @@
+Rollbar.configure do |config|
+  if Rails.env.production?
+    config.enabled = true
+    config.access_token = ENV["ROLLBAR_TOKEN"]
```

Comment:
I think there's like an `app.json` that Heroku can use to figure out what environment variables you have to make deployment easier but since I only have this one I don't think I need to add that yet.